src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border-radius: 5px;
+  border: solid 1px #f3f3f3;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $main-text-color;
+
+  &__image {
+    height: 134px;
+    object-fit: contain;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__ratings {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$main-text-color};
```

Comment:
Is it really necessary to interpolate for color? 
Maybe it will work if you just substitute a variable?

Reply:
Not fixed